lib/xsdParser.coffee

```diff
@@ -326,8 +330,12 @@ module.exports =
         # Copy fields from base
         linkType = @types[extenType.$.base]
         if not linkType
-          atom.notifications.addError "can't find base type " + extenType.$.base
-          continue
+          # Ingore link type for simple types reffering to standard simple types like xs:string, etc.
+          if type.xsdType == 'simple' and ":" in extenType.$.base
```

Comment:
What do you think about creating a global list of built-in types? Then we can check here against that list (and also in #48). As a start we can add to the list the most common built-in types:
* xs:string
* xs:decimal
* xs:integer
* xs:boolean
* xs:date
* xs:time

Reply:
I agree that the way I distinguish between a built-in type and custom type is not perfect, but it is working fine for the usual cases. So I think it is a reasonable simplification.

If you want to include list of built-in types than I would suggest to make the [list complete](https://www.w3.org/TR/xmlschema-2/#built-in-primitive-datatypes). To be perfect the namespace prefix would have to be checked too to distinguish between `xs:string` and `my:string`.

Reply:
Make sense, it will be implemented from a different issue from #49.